CHANGES.md

```diff
@@ -1,3 +1,10 @@
+## 0.3.1
+
+* Added beta version of linkerd dashboard version 2.0 :chart_with_upwards_trend:
```

Comment:
How would you feel about calling out the `<admin-port>/dashboard` URL explicitly as part of this bullet point?